src/nvme/types.h

```diff
@@ -3310,7 +3311,8 @@ enum nvme_cmd_effects {
 	NVME_CMD_EFFECTS_NCC		= 1 << 2,
 	NVME_CMD_EFFECTS_NIC		= 1 << 3,
 	NVME_CMD_EFFECTS_CCC		= 1 << 4,
-	NVME_CMD_EFFECTS_CSE_MASK	= 3 << 16,
+	NVME_CMD_EFFECTS_CSER_MASK	= 3 << 14,
+	NVME_CMD_EFFECTS_CSE_MASK	= 7 << 16,
```

Comment:
```NVME_CMD_EFFECTS_CSE_MASK``` update looks wrong. The bitfield lenght should stay at 3, no?

Reply:
Hi @igaw,
As per NVMe Base Spec 2.0d, Command Submission and Execution (CSE) field is of 3 bits (18:16). From which only 000b, 001b and 010b define something and 011b to 111b are reserved for now.
Older implementation implies bitfield length of 2, so I updated it to 7 to make the bitfield length 3.

Reply:
You are right and even wrote it in the commit message. 'sorry for the noise.